src/core.jl

```diff
@@ -109,6 +118,61 @@ struct Polygon <: Drawable
     vertices::Vector{Point}
 end
 
+"""
+    BoundaryFill <: AbstractPolyFillAlgorithm
+    BoundaryFill(x::Int, y::Int, fill_color::T, boundarycolor::T)
+
+    draw(img, verts, BoundaryFill(x, y, fill_color, boundary_color); connectverts)
+    draw!(img, verts, BoundaryFill(x, y, fill_color, boundary_color); connectverts)
```

Comment:
I feel `connectverts` should be documented as well... Generally, we use underscore to split the words, i.e., `connect_verts`. However, I don't know if `verts` are something that's widely used so have no idea if this is a good name.

Reply:
I think `connect_vertices` would be more clear but looks wordy. And it's an important parameter because boundary fill won't know when to stop without connected vertices, flood fill and scan fill don't really care about connected vertices.

Reply:
> I think `connect_vertices` would be more clear but looks wordy. And it's an important parameter because boundary fill

This looks like a good name to me. Will it be too often that the user needs to change the default behavior, i.e., `connect_vertices=false`? If `false` is rarely used, then I think it's totally fine to use some long but clear words.

Does `connect_vertices` mean the fill range is `closed` region? If so, maybe we can use `closed` as a keyword.

Reply:
`connect_vertices` won't be changed much and yes it means the fill range will be closed region. I think `closed` would be good too. Flood fill implementation would make things more clear for me, since it approaches things differently and then fitting them all in a single API would be interesting.

Reply:
`closed` looks like a better word for `draw` because it's declarative. Users might not care about implementation details(e.g., how to make it a closed range with `connect_vertices`). If other algorithms have their own way to make a closed region, the same keyword `closed` can be reused then.